src/sync/queries.ts

```diff
@@ -62,9 +121,15 @@ export const getCommits = `query ($owner: String!, $repo: String!, $per_page: In
     }
   }`;
 
-export const getBranches = `query ($owner: String!, $repo: String!, $per_page: Int!, $cursor: String) {
+export const getBranches = `query ($owner: String!, $repo: String!, $cursor: String) {
     repository(owner: $owner, name: $repo) {
-      refs(first: $per_page, refPrefix: "refs/heads/", after: $cursor) {
+      refs(first: 20, refPrefix: "refs/heads/", after: $cursor) {
```

Comment:
Can't we use 100 here, too, like in the other entities? Probably wouldn't do much for the overall sync time, but still ... .

I was going to propose to go even higher than 100, but I understand that's the maximum GitHub allows.

Reply:
I tried, but I kept getting issues because of the history of 100.  A hard graphql crash, I'm not sure why but it's definitely something internal to the API.  Reducing it to 20 * 100 made it much more stable.